modules/programs/zsh.nix

```diff
@@ -22,6 +22,10 @@ let
     mapAttrsToList (k: v: "alias -g ${k}=${lib.escapeShellArg v}") cfg.shellGlobalAliases
   );
 
+  dirHashesStr = concatStringsSep "\n" (
+    mapAttrsToList (k: v: "hash -d ${k}=${v}") cfg.dirHashes
```

Comment:
Would this need any sort of escaping like `lib.escapeShellArg`? e.g. a folder with a space in its name?

Reply:
Adding this behavior to an existing zsh test case is optional but a nice plus :+1:

Reply:
Thank you, I incorporated your suggestion, and also updated example configuration for the same, as `$HOME` in previous example configuration was getting quoted.

Reply:
But then you cannot have environment variables... I didn't think about this.
Enclosing the value in double quotes would be the better solution in the end: `''hash -d ${k}="${v}"''`.
What do you think?

Reply:
Yes, that indeed seems better. I've pushed the change, and reverted example configuration back to what it was.

Thanks!